rules/falco_rules.yaml

```diff
@@ -2428,6 +2437,12 @@
     and package_mgmt_procs
     and not package_mgmt_ancestor_procs
     and not user_known_package_manager_in_container
+  exceptions:
```

Comment:
It looks like these changes still contain exceptions, which as @leogr mentioned are not supported by the community in the default rulesets. Moreover, your branch probably needs to be rebased due to some recent changes on master.

Reply:
@jasondellaluce Thanks , This has been fixed now